notebooks/basics/single-systems.ipynb

```diff
@@ -88,26 +85,26 @@
     "### 1.1 Classical states and probability vectors <a id='classical-states'></a>\n",
     "\n",
     "Suppose that we have a [system](gloss:system) that stores information.\n",
-    "More specifically, we will assume that this system can be in one of a finite number of *classical states* at each instant.\n",
-    "Here, the term classical state should be understood in intuitive terms, as a configuration that can be recognized and described unambiguously.\n",
+    "More specifically, let's assume that this system can be in one of a finite number of *classical states* at each instant.\n",
+    "Here, the term \"classical state\" should be understood in intuitive terms, as a configuration that can be recognized and described unambiguously.\n",
     "\n",
-    "The archetypal example, which we will come back to repeatedly, is that of a bit, which is a system whose classical states are 0 and 1.\n",
+    "The archetypal example, which we will come back to repeatedly, is that of a *bit*, which is a system whose classical states are 0 and 1. Any system with two states provides a realization of a classical bit: the labels are unimportant.\n",
```

Comment:
I'd rather not muddy the waters with the sentence that starts "Any system with two states..."